src/main.py

```diff
@@ -399,6 +402,21 @@ def get_arg_parser():
         required=True,
         help="Specify the base patch version for which the package staleness report needs to be " "generated.",
     )
+    package_size_parser = subparsers.add_parser(
+        "generate-size-report",
+        help="Generates package size report for each of the packages in the given " "image version.",
+    )
+    package_size_parser.set_defaults(func=generate_package_size_report)
+    package_size_parser.add_argument(
+        "--base-patch-version",
+        required=False,
+        help="Specify the base patch version for which the package size report needs to be " "generated.",
```

Comment:
do we need this? 
can't we infer this from source-version.txt ??

Reply:
It makes sense! I'll remove this and use the base version in source-version.txt